package.json

```diff
@@ -110,8 +110,6 @@
   },
   "jest": {
     "modulePathIgnorePatterns": [
-      "/.module-cache/",
```

Comment:
This was relevant for Jest 0.1.5 which was ages ago, and @zpao wasn't sure if it was necessary. Let's live without it and see how it goes.